supporting_scripts/course-scripts/quick-course-setup/manage_programming_exercise.py

```diff
@@ -0,0 +1,68 @@
+import sys
+from logging_config import logging
+from typing import List, Dict, Any
```

Comment:
Do you use `List` here at all?

```suggestion
from typing import Dict, Any
```